app/controllers/katello/concerns/api/v2/content_overrides_controller.rb

```diff
@@ -0,0 +1,30 @@
+module Katello
+  module Concerns
+    module Api::V2::ContentOverridesController
+      extend ActiveSupport::Concern
+
+      # overriden_object => pass it either an activation key or content host
+      def validate_content_overrides_enabled(content_params, overriden_object)
+        valid_values = ["yes", "no", "0", "1", "default"]
```

Comment:
@ehelms Dont know why your comments on these got removed.
All I was trying to say was this no different than saying "ForemanCast::to_bool" other than the fact there is a "default" also. Feels like something very specific to controller data param massaging. If want it to be a constant I can just declare it in the controller.